gxa/src/main/webapp/WEB-INF/jsp/tiles/global-masthead.jsp

```diff
@@ -2,3 +2,10 @@
 
 <header id="masthead-black-bar" class="clearfix masthead-black-bar expanded">
 </header>
+
+<script>
+  // Hack that solves https://github.com/ebiwd/EBI-Framework/pull/139 (I got tired of waiting)
+  window.addEventListener('load', function(event) {
```

Comment:
Hmm the side-effect of this is that there is a delay, so it starts off unexpanded and then you can see it move (or maybe my browser is particularly sluggish today). But I can't think of a better way of doing it...

Reply:
I know, I’m aware of it. Once the assets are cached it’s very fast, although noticeable. It’s the kind of thing that people who are unaware of it will hardly ever notice.

Another option is to fork the framework but then we can’t use the CDN. Let’s hope the PR gets trhough, although it seemed it won’t because it falls outside their core design principles. I still think that if `row expanded` is a supported option, it should also be available on the global masthead. Otherwise pages look kind of unbalanced.

Anyway...